feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/data/repository/ConvictionVotingRepository.kt

```diff
@@ -45,6 +45,8 @@ interface ConvictionVotingRepository {
 
     fun ExtrinsicBuilder.vote(referendumId: ReferendumId, vote: AccountVote)
 
+    fun ExtrinsicBuilder.vote(votes: Map<ReferendumId, AccountVote>)
```

Comment:
Is there a reason to have this in interface rather then have an extension that does `forEach { vote(referendumId, vote) }`?

Reply:
Agree we can use for each here